packages/integration-tests/src/views/storagePage.ts

```diff
@@ -21,7 +21,30 @@ export const storagePage = {
     cy.get(Cypress.env('diskNumberDataLabel'))
       .should('have.length', numMasters + numWorkers)
       .each((hostDisk) => {
-        expect(hostDisk).to.contain('2');
+        expect(hostDisk).to.contain('3');
       });
   },
+  getSkipFormattingCheckbox: (hostId: string, indexSelect: number) => {
+    return cy.get(`input[id="select-formatted-${hostId}-${indexSelect}"]`);
+    ``;
+  },
+  validateSkipFormattingDisks: (hostId: string, numDisks: number, diskId: string) => {
+    cy.get(Cypress.env('skipFormattingDataLabel')).should('have.length', numDisks);
+    //Checking if checkboxes are checked/unchecked
+    storagePage.getSkipFormattingCheckbox(hostId, 0).should('not.be.checked');
+    storagePage.getSkipFormattingCheckbox(hostId, 1).should('be.checked');
+    storagePage.getSkipFormattingCheckbox(hostId, 2).should('be.checked');
+    //Checking if checkboxes are enabled/disabled
+    storagePage.getSkipFormattingCheckbox(hostId, 0).should('be.enabled');
+    storagePage.getSkipFormattingCheckbox(hostId, 1).should('be.enabled');
+    storagePage.getSkipFormattingCheckbox(hostId, 2).should('be.disabled');
+    //If a disk is skip formatting validate that warning icon is shown
+    cy.get(`[data-testid="disk-row-${diskId}"] [data-testid="disk-name"] > svg > path`)
```

Comment:
Please extract the implementation details to get the warning icon to a separate function `storagePage.getDiskSkipFormatWarningIcon()`. That makes the code easier to understand and  reusable for other tests.

As for the `M569` part, put it into a constant variable instead of as a magic number in the code.